pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -21,6 +21,10 @@
 
     private final String version;
 
+    public static String[] whitelistedPythonVersions = { "2.6", "2.7", "3.4", "3.5", "3.6" };
+    public static String defaultPython2 = "2.6";
+    public static String defaultPython3 = "3.5";
```

Comment:
I think this belongs somewhere else. Maybe the PythonExtension? I don't know where off the top of my head.

In general you don't want static variables that can mutate. This is because if the JVM keeps running, the values will be kept and you may run into issues where different builds use the same daemon which means the settings from one build may change another.

Should we specify what is meant by 3 and 2 when specified when setting the python version? Would it be best to pick the lowest from the avaliable versions that are supported?